content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -494,7 +494,8 @@ kubectl create quota test --hard=count/deployments.extensions=2,count/replicaset
 ```
 
 ```shell
-kubectl run nginx --image=nginx --replicas=2 --namespace=myspace
+kubectl create deploy nginx --image=nginx --namespace=myspace
+kubectl scale deploy nginx --replicas=2 --namespace=myspace
```

Comment:
Because this is documentation, I'd prefer to avoid the shorthand:
```suggestion
kubectl create deployment nginx --image=nginx --namespace=myspace
kubectl scale deployment nginx --replicas=2 --namespace=myspace
```